packages/sampler/stories/qa/Scroller.js

```diff
@@ -282,6 +311,12 @@ storiesOf('Scroller', module)
 			<ScrollerWithLargeContainer />
 		)
 	)
+	.add(
+		'With Focus in Container',
+		() => (
+			<ScrollerWithFocusInContainer />
```

Comment:
I believe based on the above new code, that this entire changeset could be reduced down to the following, and just replace this line (line 317) with the below code:
```
			<div>
				<Button style={{display: 'inline-block'}} small>focus to me</Button>
				<Scroller focusableScrollbar style={{height: '400px', width: '300px', display: 'inline-block'}}>
					<Item>button1</Item>
					<Item>button2</Item>
					<Item>button3</Item>
					<Item>button4</Item>
					<Item>button5</Item>
					<Item>button6</Item>
					<Item>button7</Item>
					<Item>button8</Item>
					<Item>button9</Item>
					<div>Test Test Test Test Test Test </div>
				</Scroller>
			</div>
```